include/results/Results.h

```diff
@@ -79,14 +79,10 @@ class FanResult {
  */
 class PSATResult {
 public:
```

Comment:
Remove duplicate `public` declaration. They exist on 81 and 85 in the same class and this is bad practice for C++. If the Output struct does not have any members, then do not include an output struct. Please remove it.